src/VisualStudio/VisualBasic/Impl/ProjectSystemShim/VisualBasicProject.vb

```diff
@@ -315,6 +315,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.ProjectSystemShim
 
             If Not String.IsNullOrEmpty(pCompilerOptions.wszExeName) Then
                 VisualStudioProject.AssemblyName = Path.GetFileNameWithoutExtension(pCompilerOptions.wszExeName)
+                VisualStudioProject.IntermediateOutputFilePath = Path.Combine(pCompilerOptions.wszOutputPath, pCompilerOptions.wszExeName)
```

Comment:
I wonder if it's worth adding a Not String.IsNullOrEmpty check for wszOutputPath too? Mostly for paranoia?